src/components/login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes, Component } from 'react'
+import { translate } from 'react-i18next'
+
+class Login extends Component {
+	constructor (props) {
+		super(props)
+		this._handleSubmit = this.handleSubmit.bind(this)
```

Comment:
I would strongly avoid this. 

It might seem convenient but it's basically the whole opposite of the fat arrow functions spec and might lead ton confusions. 

fat arrow functions are bound to their lexical scope. 
In this case it wouldn't be the class, but the scope in which the class is defined.

It works because Babel handles the magic for you, but it could lead to serious misunderstandings. 

[Here's an example](http://jsbin.com/kuzujisude/edit?js,console) that shows the normal behavior.